src/steps/companies/__recordings__/fetchCompanies_60647338/recording.har

```diff
@@ -0,0 +1,193 @@
+          "content": {
+            "_isBinary": true,
+            "mimeType": "application/json;charset=utf-8",
+            "size": 2811,
+            "text": "[\"1f8b0800000000000000d5595b6fa33818fd2f3c43c53dd03e65d2ee4ea44ebbea6556abd1c832c649bc2598c1a65555f5bf8fc99d2424408030ed432b87e0f31d7ffe6ee7438a314b02cea4cb1f1f5244630e83a12f5deaaa6ed886a3c912a2d30886efe9a26dd9bae1daae61ca1261d738c01c8be5110c1896a528a6118e39c1e25d1fd28481309902b114021f8b\",\"27d2c557182458ba94544996389962c6e134922e355bb735d7325ccbea19b2c46812a3f4b141ff76f07cdb7fbab99696ab290ce981064112fdb3de4f9692c88702cc97f76786e3f4a1300902597ac53123349c5b17c269fad65d6472296079100ec1fe4ec4977efcfcfcf9294b6fd86384e34d3e10f5f11b8d5fe20b1fbfe640301dd53acecddcec1d36744bb3544defed67648948ae19d0c26a59f8d8af447c7fc69c661b16563da4e806761553f33cc5433d437147ae8a2ccf1b415bdd77a04b13661ca6070819a388a40f0144430e112fe063a6dab30dad711fcbc377ccd3b2f04ef53414e3747b9875b6cc01163cdbfbbba7fee0e9314b5132a3617530870e6d3f4b1bf8e4caf072a0e4636fce2b1791c50b287a11e6762de2ad70b51befc4eee9f100e2334047000601a06fe1163f1b27773e7fcc472a5701da25cf4c3c16513e3707a4dbaca9774c4d15f0f582d43f7c03cfc3ba89df862757417794ef2cf446e30087f118730011a249c881a88d3ce89180f0f74c54b84803be6e98b6ebe8e2d2b9464e567075c7d9b0e4b17f7b9375fe9cbc7f30261cc028d70c712b1e0490f129f5c988603f273dcd5f586caf72b5d0414e7690ed24a613816598c8b8bda817c838cc25a454be6efe9266d03698bd5bbcb3698ef43122a9e5600abb98c2b7e1b59ec917251bf0de97a9b28b597c0fca3f3d83fb740a497862d3f865787d735733dd0b60d57bc76324674137d853ceec59f33b1076fc9bda71c6983bfb235782d4a5c84abdff31e25bc1623d4d3a77b858c393aba1eb509c58248ae5c401bc113e015ef24ec2318869803b97d20e206d3dbbcdf04c48904e6cd2a9272942574b23a51c78ed4e94048855579cb6c927f4918ddde62cc0663ac956eff3e2818ed75c7b50fed93597201ffa3ee1c256185ccfaa1c36df48c0e7783081e1182f56a658b4d2fd443c3d5bf8948b6829aa6babaae134a0a538a6abea9ad63d2d651b58135a4a5afb935782f085a07b2f06c7750dbd4531a51e447bfddcb330f685132916ece98ae9ea86e2f910299e659a1a44beed18e878e55b4d4d490fd3510dbda36aca36bc93d594ade1dd571c3012be907d31cd3434f16336dd1ea0ecacae04a23aba03c3efa950b56d45533d53314dac29aee7e88a3532b02a7e4796eb1ef5bd031ad5f25a14bc316d6b5425e0d593d36ab9eb223c018ed124a4011d6fd5b730826882af122f097972257a783f8a3163572915e20318013f6457630ad8af04c6d8bfe2a266e722d77b09e7342c764e4dcc20768c925bb4a986f9452d274b423f613ccec4a8e1dd5ff70fdffa4fc3fb3bf07433f87a777f7bfff77fa07f770d1e6f1ebe0f07b3197d91dbd544f85a21966b065c4774abe54c4a2ac28bf2a8f98c5a4e11de86756e45b8bde87fa222dce13c5051116e2b34545384bb1c076a5484c57dd444f3b81926baa6081f85585a119e17b07b85d7c54705a26601bf3c4d1e2e8ef2a86f16ec000b55e165e5e692a5f779e5e63a0bf1960b83927273dbf54139b9b98132a1b2dcdc6a8950556eee7079b0476eae30c56a476f2e0bac2b0ddb8ee0bc32e47cec6e0bce6520758557c240947801419be4ce06ebe7e3758d492e0da92bbc1e10f21752c6b903718e905f1c5d8722f0c9427edbc5426521bf81baa18290df9a9a5149c8af5bcca828e4b77a9b2b08f9ddbecf9585fc76075e1585fcce727f8a902f9626907da3315e69f3743462988b1d040594c340fcf7f91bfea6cc8247390000\"]"
```

Comment:
Looks like this is gzipped. There is an example of how to unzip the response text in the Azure integration (using an SDK test helper): https://github.com/JupiterOne/graph-azure/blob/master/test/helpers/recording.ts#L26